cinema/models.py

```diff
@@ -35,12 +39,20 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(instance, filename):
+    _, extension = os.path.splitext(filename)
+    return os.path.join(
+        "uploads/", f"{slugify(instance.title)}-{uuid.uuid4()}{extension}"
+    )
```

Comment:
```suggestion
    return os.path.join(
        "uploads", f"{slugify(instance.title)}-{uuid.uuid4()}{extension}"
    )
```

Read how this method works